src/app/application-layout.tsx

```diff
@@ -82,15 +84,20 @@ export default function ApplicationLayout({
             width={20}
             height={20}
           />
-          <span className="text-lg font-medium text-white">AIBTCDEV</span>
+          <Image
+            src="/logos/aibtcdev-primary-logo-white-wide-1000px.png"
+            alt="AIBTCDEV"
+            width={150}
+            height={300}
+          />
         </div>
         <Button
           variant="ghost"
           size="sm"
           onClick={() => setRightPanelOpen(!rightPanelOpen)}
           className="text-zinc-400"
         >
-          <Wallet className="h-5 w-5" />
+          <Wallet className="h-5 w-5" /> Wallet
```

Comment:
Where does this text show and do we need it? Thinking clean/minimal so not sure why we need the text next to the component.